cfgov/unprocessed/js/modules/post-filter.js

```diff
@@ -102,14 +102,30 @@ PostFilter.prototype = {
   },
 
   onSubmit: function( e ) {
-    var dateRange =
-      dateRangeFormatter.format( this.$gte.val(), this.$lte.val() );
-    if ( dateRange && dateRange.isValid ) {
+    var $form = $( e.target );
+    var fromDateValue = this.$gte.val()
+    var toDateValue = this.$lte.val()
+    var dateRange = dateRangeFormatter.format( fromDateValue, toDateValue );
+
+    if ( ( dateRange && dateRange.isValid ) ) {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/cb42c16f5d06202d9a6d172276ee2878beda77d1